### PR TITLE
(HTCONDOR-2772)  Improve Schedd.spool() documentation for version 1.

### PR DESCRIPTION
**Pull request by Todd-L-Miller**
Correct mistake in spool() docstring and add example.

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [x] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [x] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

**Comment by Todd-L-Miller** [comment]
A rendering including the change to `Submit.spool()`'s docstring is (temporarily) [available](http://azaphrael.org/indexed/html/apis/python-bindings/api/htcondor.html#htcondor.Schedd.spool).

File: docs/apis/python-bindings/api/htcondor.rst [hunk 1]
```diff
@@ -42,6 +42,31 @@ Interacting with Schedulers
...
+            "output": "hostname.out",       # anything the job prints to standard output will end up in this file
+            "error": "hostname.err",        # anything the job prints to standard error will end up in this file
+            "log": "hostname.log",          # this file will contain a record of what happened to the job
+            "request_cpus": "1",            # how many CPU cores we want
+            "request_memory": "128MB",      # how much memory we want
+            "request_disk": "128MB",        # how much disk space we want
+            "transfer_input_files": "hello.txt",
+            })
+
+        sad = coll.query(htcondor.AdTypes.Schedd, constraint=f"Name=?={test}",projection=["Name", "MyAddress", "DaemonCoreDutyCycle"])[0]
```
Comment (ColeBollig) [inline docs/apis/python-bindings/api/htcondor.rst:65]: Do we need `DaemonCoreDutyCycle` in the projection? Here

File: docs/apis/python-bindings/api/htcondor.rst [hunk 2]
```diff
@@ -42,6 +42,31 @@ Interacting with Schedulers
...
    .. automethod:: spool
+
+   A fully-worked example follows.
+
+   .. code-block:: python
+
+        import htcondor
+
+        coll = htcondor.Collector("cm.pool.tld")
+        test = htcondor.classad.quote("remote-schedd.pool.tld")
```
Comment (ColeBollig) [inline docs/apis/python-bindings/api/htcondor.rst:53]: Do we want to call this temp variable `test` (also referred later in collector query constraint)... I don't care enough to make you change this but what wanted to point it out

File: docs/apis/python-bindings/api/htcondor.rst (hunk 1 above)
Comment (Todd-L-Miller) [reply docs/apis/python-bindings/api/htcondor.rst:65]: No, I just didn't look at the example closely enough.

File: docs/apis/python-bindings/api/htcondor.rst (hunk 2 above)
Comment (Todd-L-Miller) [reply docs/apis/python-bindings/api/htcondor.rst]: Renamed to `quoted_schedd_name` for lack of anything better.

**Review by ColeBollig — APPROVED** [review]
LGTM

**Pull request merged**